msteams-platform/tabs/design/tabs.md

```diff
@@ -138,6 +138,8 @@ Notify users of tab activity one of the following ways:
 
 ## Best practices
 
+These recommendations can help you create a quality app experience. While there may be some overlap, you should also make sure your app adheres to the [Teams store validation guidelines](~/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md).
```

Comment:
@ramendhe Do we want to say something like this to clear up confusion around design best practices and validation guidelines?